src/server/index.jsx

```diff
@@ -87,6 +88,16 @@ server
       }
     });
   })
+  .get(manifestRegexPath, async ({ params }, res) => {
+    const { service } = params;
+    const manifestPath = `${__dirname}/public/${service}/manifest.json`;
```

Comment:
```suggestion
    const manifestPath = `./${publicDirectory}/${service}/manifest.json`;
```

Does this need to be the absolute path that `__dirname`returns or could it just be `./build/public`?

Reply:
`res.sendFile` wants an absolute path and not a relative one. 
See the error when using a relative path: `(node:46712) UnhandledPromiseRejectionWarning: TypeError: path must be absolute or specify root to res.sendFile`